### PR TITLE
add resetOnIdle method + timer 60sec

### PR DESCRIPTION
**Pull request by igor-kamil**

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -5,14 +5,22 @@
...
-      scrollbar-width: none; /* Firefox */
-    }
+  html,
+  body {
+    touch-action: pan-x pan-y;
   }
-  
\ No newline at end of file
+
+  .allow-pinch-zoom {
```
Comment (FrantisekMichalSebestyen) [inline assets/css/main.css:13]: is this needed?

File: pages/publication/[id].vue [hunk 2]
```diff
@@ -21,7 +22,7 @@
           ref="booksRefs"
           :book-index="index"
         >
-          <div class="h-[650px]">
+          <div class="h-[650px] allow-pinch-zoom">
```
Comment (FrantisekMichalSebestyen) [inline pages/publication/[id].vue:25]: is this needed?

**Review by FrantisekMichalSebestyen — CHANGES_REQUESTED** [review]

File: assets/css/main.css (hunk 1 above)
Comment (igor-kamil) [reply assets/css/main.css:13]: already removed in https://github.com/SlovakNationalGallery/stex-reader/pull/6

File: pages/publication/[id].vue (hunk 2 above)
Comment (igor-kamil) [reply pages/publication/[id].vue:25]: no :D

**Review by FrantisekMichalSebestyen — APPROVED** [review]

**Pull request merged**